tools/build_file_generator/build_file_generator.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package com.google.startupos.tools.buildfilegenerator;
+
+option java_package = "com.google.startupos.tools.buildfilegenerator";
+option java_outer_classname = "Protos";
+
+message Import {
+  string root_dir = 1;
```

Comment:
Can you explain what these fields are with an example?

Reply:
Actually, `root_dir` field doesn't make sense, deleted it.
For example, this class:

```
package com.google.startupos.tools.buildfilegenerator;

import com.google.startupos.common.flags.Flag;
import com.google.startupos.common.flags.FlagDesc;
import com.google.startupos.common.flags.Flags;

import java.util.Arrays;
import java.util.List;

public class JavaClass {
  @FlagDesc(name = "file_path", description = "Path to file")
  static Flag<String> filePath = Flag.create("~/JavaClass.java");

  public static void main(String[] args) {
    Flags.parseCurrentPackage(args);

    List<String> classes = Arrays.asList(filePath.get());
  }
}
```

will be parsed:

```
classname: "JavaClass"
package: "com.google.startupos.tools.buildfilegenerator"
import {
  import_dir: "com"
  import_dir: "google"
  import_dir: "startupos"
  import_dir: "common"
  import_dir: "flags"
  imported_class_name: "Flag"
}
import {
  import_dir: "com"
  import_dir: "google"
  import_dir: "startupos"
  import_dir: "common"
  import_dir: "flags"
  imported_class_name: "FlagDesc"
}
import {
  import_dir: "com"
  import_dir: "google"
  import_dir: "startupos"
  import_dir: "common"
  import_dir: "flags"
  imported_class_name: "Flags"
}
import {
  import_dir: "java"
  import_dir: "util"
  imported_class_name: "Arrays"
  standard_java_package: true
}
import {
  import_dir: "java"
  import_dir: "util"
  imported_class_name: "List"
  standard_java_package: true
}
has_main_method: true
```

Reply:
Added the comments to the proto file.

Reply:
I don't think we need the split to `import_dir`.
How about we use this structure:
```

Reply:
The code is changed to work with the new structure.